src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
```

Comment:
> navigation__items-list__item - not valid BEM name
 
Try to split into new BEM block like.  
```suggestion
        <ul class="navigation__items">
          <li class="nav-item">
            <a href="#" class=nav-item__link is-active">Apple</a>
          </li>
```